portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -14,19 +14,34 @@
 
 package com.google.sps.servlets;
 
+
 import java.io.IOException;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import com.google.gson.Gson;
+import java.util.ArrayList;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
-  @Override
-  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;"); 
-    response.getWriter().println("<h1>Hello Erik !</h1>");
-  }
+    ArrayList<String> mylist = new ArrayList<String>();
```

Comment:
Also can you change the name myList to something more descriptive. I'm assuming this is for your comment page? If so maybe something like commentList.

Reply:
Good Catch, I almost for got about that. Will change right now.